### PR TITLE
merge dsl2 into dev

### PR DESCRIPTION
**Pull request by maxulysse**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
    - [ ] If you've added a new tool - add to the software_versions process and a regex to `scrape_software_versions.py`
    - [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/sarek/tree/master/.github/CONTRIBUTING.md)
    - [ ] If necessary, also make a PR on the nf-core/sarek _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint .`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

**Comment by maxulysse** [comment]
Once approved and merged, we can change the base branch for https://github.com/nf-core/sarek/pull/412 and delete the `dsl2` branch.

File: conf/genomes.config [hunk 1]
```diff
@@ -8,76 +8,91 @@
...
+            intervals               = "${params.genomes_base}/wgs_calling_regions_Sarek.list"
+            known_indels            = "${params.genomes_base}/{1000G_phase1,Mills_and_1000G_gold_standard}.indels.b37.vcf"
+            known_indels_index      = "${params.genomes_base}/{1000G_phase1,Mills_and_1000G_gold_standard}.indels.b37.vcf.idx"
+            mappability             = "${params.genomes_base}/out100m2_hg19.gem"
+            snpeff_db               = 'GRCh37.75'
+            vep_genome              = 'GRCh37'
+            vep_species             = 'homo_sapiens'
+            vep_cache_version       = '104'
+        }
+        'GRCh38' {
```
Comment (FriederikeHanssen) [inline conf/genomes.config:34]: Do we not need to update the keysn here too?

**Review by FriederikeHanssen — APPROVED** [review]
Looks good to me. Massive PR, but most has been reviewed before in the dsl2 branch.

File: conf/genomes.config (hunk 1 above)
Comment (maxulysse) [reply conf/genomes.config:34]: which ones were not updated?

**Pull request merged**